src/Api.ts

```diff
@@ -16,6 +17,16 @@ async function postRequest<T>(url: string, bodyObject: {}) {
     })
     return await res.json() as T
 }
+async function uploadFile<T>(url: string, keyName : string, file: File) {
```

Comment:
de param url kan hier wel een lokale var worden, anders moet je telkens diezelfde url in de functie meegeven

Reply:
Je kunt ook andere bestanden uploaden, mocht dat later nog komen